.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature Request
+about: Template for a question around the topic of deploying Wire 
+title: Request: [REQUEST NAME]
+labels: 
+assignees:
+---
+
+### Basic information
+
+* On-premises: Yes/No
+* Cloud-Provider (e.g. AWS, GCP, etc): TODO
+* Installation type: demo/prod
+* Kubernetes version: TODO
+* related Technology + Version (Ansible, Helm, Terraform): TODO
```

Comment:
This section makes no sense to me in the feature request section, as I would assume it's rarely related/helpful to fill this out. Remove this section?